pep/app/pods/components/home/component.ts

```diff
@@ -122,19 +126,28 @@ export default class Home extends Component<HomeArgs> {
     }
 
     /**
-     * Returns the expert pick of the day abstract
+     *Returns the expert pick of the day abstract
+     *
+     * @return {*}  {Promise<void>}
+     * @memberof Home
      */
     @action
-    async loadModel() {
+    async loadModel(): Promise<void> {
         const expertPicks = this.configuration.base.home.expertPicks;
-        const result = await this.store.findRecord('abstract', expertPicks[expertPicks.length - 1].articleId);
+        const specialDate = this.configuration.base.home.expertPicksStartDate;
+
+        const dayDifferential = moment(new Date()).diff(specialDate, 'days');
+        const index = dayDifferential % expertPicks.length;
```

Comment:
like the `expertPick()` getter i _think_ you could bundle up the `index` calculation into its own getter too, and would also remove the need for the extra `@tracked` currentlyShownIndex value, since its purely derived state, doesnt need to be mutable:

```ts
get expertPickIndex() {
    const expertPicks = this.configuration.base.home.expertPicks;
    const specialDate = this.configuration.base.home.expertPicksStartDate;
    const dayDifferential = moment(new Date()).diff(specialDate, 'days');
    return  dayDifferential % expertPicks.length;
}

get expertPick() {
    const expertPicks = this.configuration.base.home.expertPicks;
    return expertPicks[this.expertPickIndex];
}
```